.github/workflows/_get_app_metadata.yml

```diff
@@ -27,6 +27,16 @@ on:
         required: false
         default: 'None'
         type: string
+      flags:
+        description: "Additional compilation flags (default to none)"
+        required: false
+        default: ''
+        type: string
+      use_case:
+        description: "The use case to build the application for"
```

Comment:
description: "The use case to build the application for. Ignored if 'flags' is set"